### PR TITLE
LPS-186596 and LPS-185983 Adds Portlet to render the view and a resource command to generate text using openAI API

### PR DESCRIPTION
**Pull request by lfbesada**
https://issues.liferay.com/browse/LPS-186596
https://issues.liferay.com/browse/LPS-185983

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:stop

**Comment by lfbesada** [comment]
ci:stop

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

File: modules/apps/ai-creator/ai-creator-openai-web/src/main/java/com/liferay/ai/creator/openai/web/internal/editor/configuration/AICreatorOpenAIEditorConfigContributor.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+			!_isAICreatorOpenAIGroupEnabled(
+				themeDisplay.getCompanyId(), themeDisplay.getScopeGroupId())) {
+
+			return;
+		}
+
+		jsonObject.put(
+			"aiCreatorOpenAIURL",
+			() -> {
+				PortletURL portletURL =
```
Comment (ealonso) [inline modules/apps/ai-creator/ai-creator-openai-web/src/main/java/com/liferay/ai/creator/openai/web/internal/editor/configuration/AICreatorOpenAIEditorConfigContributor.java:72]: @lfbesada Could it be possible use PortletURLBuilder?

Comment (lfbesada) [reply modules/apps/ai-creator/ai-creator-openai-web/src/main/java/com/liferay/ai/creator/openai/web/internal/editor/configuration/AICreatorOpenAIEditorConfigContributor.java]: Hi @ealonso
Thanks for the tip! I have added a commit to change it and rebase the branch solving the regeneration the buildLand to solve the conflicts.
Could you please review it?
Thanks in advance!

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**